external-docs/docs/guides/configuration/templates.md

```diff
@@ -80,3 +80,89 @@ Units used, have to match the terminal part of the URI exactly including case se
 | Template                                                                       | Description                                    | Example                                                                              |
 |--------------------------------------------------------------------------------|------------------------------------------------|--------------------------------------------------------------------------------------|
 | [qudt-units](https://purl.org/csv-cubed/qube-config/templates/qudt-units.json) | A template which contains units of measurement | [http://qudt.org/vocab/unit/PoundSterling](http://qudt.org/vocab/unit/PoundSterling) |
+
+## How to deal with Mixed Periods
+
+The purpose of this section is to provide guidance on configuration of a data set with mixed time periods, e.g. daily, weekly, monthly, government-year, half-year, etc. Also, an example of how to wrangle a dataset using Python so that the CSV can be used to create a CSV-W is provided.
+
+In the table below we have two different types of dates, representing daily and weekly frequencies. The format of the daily figures is acceptable. However, there is an issue with the weekly data. The date `2019-04-01` represents the week beginning `2019-04-01`, and `2019-04-08` represents the following week. Instead, the ISO8601 format `week/YYYY-WWW` should be used, e.g., the week beginning `2019-04-01` should be `week/2019-W14`.
+
+| Period     | Frequency | ... |
+|------------|-----------|-----|
+| 2019-04-01 | Daily     | ... |
+| 2019-04-02 | Daily     | ... |
+| 2019-04-01 | Weekly    | ... |
+| 2019-04-08 | Weekly    | ... |
+
+**NB:** it is important to validate whether the weekly data matches the ISO8601 definition, as this is slightly different from the traditional Gregorian calendar.
+
+**Things to note:**
+
+- The week date representation must commence on a Monday. If not, a [gregorian-interval](https://purl.org/csv-cubed/qube-config/templates/gregorian-interval.json) of one week must be used.
+- Unlike the Gregorian calendar, there are no weeks that extend across years. Each ISO8601 year consists of either 52 or 53 weeks depending on when the year begins. If 1 January falls on a Monday, Tuesday, Wednesday or Thursday it is in week 01 of the new year. If 1 January falls on a Friday, Saturday or Sunday, it is in week 52 or 53 of the previous year.
+- More information can be found [here](https://en.wikipedia.org/wiki/ISO_8601#Week_dates)
+
+### How to format mixed periods
+
+- For daily data, prefix the date with `day/`: `day/YYYY-MM-DD`
+- For weekly data, calculate the ISO8601 week value and prefix with `week/`: `week/YYYY-WWW`. For example, the week commencing 01/04/2019 would be `week/2019-W14`.
+- Once the dates have been formatted as above, the 'Frequency' column should be removed.
+
+### Example Python code
+
+The following code is an example of how this can be achieved in Python:
```

Comment:
I know to us the imports make it clear what needs to be `pip install`ed for this to work, but can you add something that tells them they need to have click, pandas and numpy installed? 

Also, the pathlib import should be in a separate section *above* the third-party imports by convention.

Reply:
If csvcubed is installed all the dependencies are available in the same environment. We use click, and pathlib is core.